src/routes/login/components/LoginForm.tsx

```diff
@@ -75,6 +75,29 @@ const LoginForm = (): JSX.Element => {
                 <Text fontWeight="semibold" fontSize="3xl" marginBottom="4">
                     Login to {organizationId}
                 </Text>
+
+                {authInfo?.oidc && (
+                    <div>
+                        <a
```

Comment:
Shoud use wouter `<Link>` for internal links

Reply:
This is an external link to the backend, but I can use chakra's